uid/snowid.go

```diff
@@ -74,7 +74,7 @@ func NewNode(node int64) (*Node, error) {
 
 	var curTime = time.Now()
 	// add time.Duration to curTime to make sure we use the monotonic clock if available
-	n.epoch = curTime.Add(time.Unix(epoch/1000, (epoch%1000)*1000000).Sub(curTime))
+	n.epoch = curTime.Add(epoch.Sub(curTime))
```

Comment:
Previously the epoch was an int. We used a `time.Date` to define it (which is much nicer). So now we can use the date directly instead of converting it to an int and back to a date.